torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1470,7 +1470,7 @@ void TensorExprKernel::bindInput(const torch::jit::Value* input) {
                 for (size_t i = 0; i < axes.size(); i++) {
                   idx = idx + axes[i] * IntImm::make(*strides[i]);
                 }
-                return inBuffer(idx);
+                return inBuffer.load(idx);
```

Comment:
Is `operator()` no longer supported for indexing into a buffer?

Reply:
Correct, I removed it. It won't be as convenient anyway once we make `Placeholder`s mem-managed and they all become pointers. If we really need this functionality I think we might need to introduce somtheing like TensorHandle and PlaceholderHandle (it might looks ugly, but it'd be consistent with the rest of the system).